### ISSUE TITLE
Amend "Advanced settings for shareable links" section

### ISSUE THREAD

**Issue by DennisGaida** [MicrosoftDocs/OfficeDocs-SharePoint#245]
The section "Advanced settings for shareable links" currently says:
*"If your external sharing setting for SharePoint is set to allow shareable links (even if external sharing is currently turned off), you can set an expiration time for those links by choosing a value from the Links expire within drop-down list. You can also choose whether shareable links can give people permission to edit shared files and folders."*

The explanation given is not complete and also erroneous (the dropdown also gives permission to upload files). It should explain the the three existing configuration options in detail, namely it should include the following information:
* **Links must expire within this number of days**: Links will expire within the given timeframe for all users. If set to zero (0) there is no expiration on links.
  * *Can this be overruled by users?*
* **Files**: Can be set to "View" and "View, edit and upload".
  * *"View, edit and upload" should be explained*
* **Folders**: Can be set to "View" and "View, edit and upload".
  * *"View, edit and upload" should be explained*

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 40a570bb-2567-565c-e9ff-bf1ccbf38d0e
* Version Independent ID: 3d342dea-9dd3-008c-20c6-902452302e95
* Content: [Manage sharing in OneDrive and SharePoint](https://docs.microsoft.com/en-us/onedrive/manage-sharing?redirectSourcePath=%252farticle%252fee8b91c5-05ec-44c2-9796-78fa27ec8425)
* Content Source: [OneDrive/manage-sharing.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/OneDrive/manage-sharing.md)
* Service: **one-drive**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

**Comment by AndreaBarr** [comment]
@DennisGaida Thank you for this feedback

I will get this over to the SharePoint team for investigation and updating. Thank you for taking time to make the docs better. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
